### DISCUSSION TITLE
Allow to request a pull request review from non-organization members

### DISCUSSION THREAD

**Discussion by driesvints** [community/community#4586] [Pull Requests]
At the moment it is only possible to request reviews from organization members or external collaborators that have been added to the repository. However, numerous times, while working on pull requests for @laravel, I've found myself in wanting to request a review from a community member who is highly knowledgable about a certain topic or other reasons. 

For example, we have a community member @JosephSilber who is highly knowledgable about the Collections component of `laravel/framework`. Because he's not an organization member I need to ping him manually on every PR that involves the collection component. It would be more useful to request a review from them. 

Additionally, it would be nice if files like `CODEOWNERS` worked with non-organization members. To iterate on the above example, we've set up such a file to automatically request a review from Joseph when changes are made to the Collections component. 

https://github.com/laravel/framework/blob/8.x/.github/CODEOWNERS

Except, this doesn't work because he's a non-organization member. 

I also posted another relevant discussion: https://github.com/github/feedback/discussions/4395. In combination with that, I'd expect that if a repository was limited to only accept pull request approvals from organization members or external collaborators, I'd also expect non-organization members who explicitly were requested a review to be able to approve pull requests.

**Comment by Aurorum** [comment]
Strong supporter of this proposal! Not sure if this would justify creating a new discussion, but given that the two are linked, I feel like it would also be beneficial to allow regular external contributors to request reviews from those within an organisation too. For a limited period of time, this was possible until it was removed in around February 2020. I don't believe a reason was ever given for its removal, but it used to be really helpful for me.

**Comment by jrzyshr** [comment]
This discussion was started almost 9 months ago.  Does anyone from the GitHub team have anything to add here?  For open source projects that involve contributions from folks in the community across organizations, I think this would be an important feature to enable CODEOWNERS to be from outside an organization.

For example, I help manage an open source project called "What The Hack" at Microsoft.  It is a collection of hackathons/courseware authored by contributors from within and outside of the company.  All contributions come via PRs and only a small team of admins have "write" access to the repo.  We have over 80 contributors.  We would like the original authors of each hackathon to be requested for review if a PR comes in updating their hack.  

Our challenge is, most of our authors are not part of the organization, and thus cannot be added to the CODEOWNERS file.  It seems as though this feature would be something the open source community would want?

**Comment by jbytheway** [comment]
It's another year since the last comment, so I wanted to add my support and mention that this is also a problem faced by an open source project I contribute to.  For example [this PR](https://github.com/CleverRaven/Cataclysm-DDA/pull/65838) was our latest (failed) attempt to find a workaround for this issue.

Similar to the project @jrzyshr described above, we have many small parts of the project that are the responsibility of different contributors, many of whom are not organization members.  It would be very useful to allow them to get notified, via `CODEOWNERS` or some other mechanism.  Already five different non-members have added themselves to the file, not realizing that it would be ineffective.
